### ISSUE TITLE
[exporter/azuredataexplorer] Accept exporterhelper config for azure data explorer exporter 

### ISSUE THREAD

**Issue by david-esteves-sh** [open-telemetry/opentelemetry-collector-contrib#24329]
### Component(s)

exporter/azuredataexplorer

### Is your feature request related to a problem? Please describe.

I am currently using the AzureDataExplorer exporter from the opentelemetry-collector-contrib and do not know of a way to configure the throughput or persistence with the current configuration. The current implementation already uses the ExportHelper but the `queue_settings` values are not configurable. I would like to be able to tweak the configuration to maximize throughput and also use the storage queue options to prevent data loss on shutdown.

### Describe the solution you'd like

Allow to pass in `sending_queue`, `retry_on_failure` and `timeout` config values from the [export helper](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md) as supported in many other exporters (e.g. otlpexporter, kafkaexporter, etc). Either for the entire exporter or per telemetry type, we could have it configurable by for metrics, logs, traces all separately as a lot of the config today seems to be

```
exporters:
  azuredataexplorer:
    cluster_uri: "https://CLUSTER.kusto.windows.net"
    application_id: "f80da32c-108c-415c-a19e-643f461a677a"
    application_key: "xx-xx-xx-xx"
    db_name: "oteldb"
    traces_table_name: "OTELTraces"
    # Newly acceptable config values
    retry_on_failure:
      enabled: false
    sending_queue: 
      storage: file_storage/otc
      num_consumers: 25
      queue_size: 30000    
```
or 
```
exporters:
  azuredataexplorer:
    cluster_uri: "https://CLUSTER.kusto.windows.net"
    application_id: "f80da32c-108c-415c-a19e-643f461a677a"
    application_key: "xx-xx-xx-xx"
    db_name: "oteldb"
    traces_table_name: "OTELTraces"
    # Newly acceptable config values
    retry_on_failure:
      enabled: false
    traces_sending_queue: 
      storage: file_storage/otc
      num_consumers: 25
      queue_size: 30000    
    metrics_sending_queue: 
      num_consumers: 5
      queue_size: 300  
```

**Comment by ag-ramachandran** [comment]
Hello @david-esteves-sh 
We'll evaluate this and see how we can prioritize this for one of the upcoming releases

cc: @asaharn

**Comment by crobert-1** [comment]
Removing `needs triage`, `stale`, and assigning to @asaharn as a PR has been opened.

**Comment by ag-ramachandran** [comment]
Hi @crobert-1 , This PR is now merged. Can this issue be closed!

**Comment by crobert-1** [comment]
Absolutely, thanks for letting me know!

Fixed by #27272
